itsnameissteven/Brew-by-Design
# Feature/route by search

## Is this a fix or a feature?
This is a feature
## What is the change?
The user is now able to search for beers
## What does it fix?
n/a
## Where should the reviewer start?
Review the new search bar, and SearchResults component.
## How should this be tested?
Search for a beer in the browser. When matched a list a beers should appear on the search page.
## Relevant ScreenShots
